docs/changelog/0007-reorganize-pillows.md

```diff
@@ -0,0 +1,106 @@
+# 7. Reorganize pillows
+
+**Date:** 2018-11-26
+
+**Optional per env:** No
+
+## CommCare Version Dependency
+This version of CommCare must be deployed before rolling out this change:
+[todo](https://github.com/dimagi/commcare-hq/commit/todo)
+
+
+## Change Context
+Pillows read changes from kafka and do various processing such as sending  to
+elasticsearch, transforming into a UCR table row etc. A doc for same change is read
+multiple times for each processor, since there are pillows for each processor.
+This is inefficient, so we have combined multiple processors that apply for a
+given document type (also called `KAFKA_TOPIC`) such as form/case/user under
+one pillow. For e.g. A new single `get_ucr_es_case_pillow` pillow replaces
```

Comment:
I'm confused by the use of `get_*` naming here. Looking at the commcare-hq PR, I would have thought this pillow is called `kafka-case-ucr-es`. Is that right? If so can you update this and the Old Pillow / New Pillow table below to use the pillow names rather than the names of the functions that build them? Someone reading this will then be able to use that to reorganize their app-processes.yml.

Reply:
Great point, will update. The naming confusion is due to the fact that there is a pillow-name, kind of slug and then there is an actual pillow getter that the slug points to, with no obvious formatting rules.